requirements-testing.txt

```diff
@@ -2,6 +2,4 @@ pytest
 pytest-cov
 ndx-events>=0.2.0  # for special tests to ensure load_namespaces is set to allow NWBFile load at all timess
 parameterized>=0.8.1
-scikit-learn  # For SI Waveform tests
-numba; python_version <= '3.10'  # For SI Waveform tests
```

Comment:
I was proposing just moving `spikeinterface[qualitymetrics]>=0.99.1` here if that's possible